source/unified-test-format/tests/invalid/expectedTracingSpans-missingPropertyClient.yml

```diff
@@ -0,0 +1,13 @@
+description: "expectedTracingSpans-missingPropertyClient"
+
+schemaVersion: "1.26"
+
+createEntities:
+  - client:
+      id: &client0 "client0"
+
+tests:
+  - description: "missing required property client"
+    operations: []
+    expectTracingMessages:
+      spans: [{}]
```

Comment:
Same as above. This should have a valid span object to ensure a missing `client` is actually what causes a validation error.